benchmarks/grain_size_plunge/README.md

```diff
@@ -0,0 +1,12 @@
+# Grain Size Plunge at Passive Margins
+
+This is a benchmark case that reproduces the model in Fig. 4 of
+
+Mulyukova, E., & Bercovici, D. (2018). Collapse of passive margins by
+lithospheric damage and plunging grain size. Earth and Planetary Science
+Letters, 484, 341-352.
+
+It models the grain size evolution under constant stress in a passive margin.
+No analytical expression for the the time-evolution of the grain size exists, but
+plotting the evolution using the 'plot.plt' script produces a figure very close to
+Fig. 4.
```

Comment:
Did you want to add the notebook that is relevant to this benchmark as well? I think that would be useful.